k8s/mongodb-backup-agent/mongo-backup-dep.yaml

```diff
@@ -41,6 +41,9 @@ spec:
             memory: 768Mi
         volumeMounts:
         - name: mdb-bak-certs
+          mountPath: /etc/mongod/ca/
+          readOnly: true
+        - name: mdb-ca
```

Comment:
This will fail tests as the `secretName` (mentioned below) should be the `name` here.